.circleci/config.yml

```diff
@@ -219,3 +219,16 @@ jobs:
             ssh-add ~/.ssh/id_rsa_* ~/.ssh/id_rsa
           fi
           ./script/chart_sync.sh kubernetes-bitnami kubernetes@bitnami.com
+  push_images:
+    machine: true
```

Comment:
Out of curiosity, why do we need to set machine: true here? Is it so we have docker installed?

Also, can this step be skipped all together even before being provisioned (if it is not being done already)

What I mean is that this step should be only provisioned and run if -z "$CIRCLE_PULL_REQUEST"